server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/core/Capability.scala

```diff
@@ -43,6 +43,7 @@ object CapabilityIdentifier {
   val EMAIL_SUBMISSION: CapabilityIdentifier = "urn:ietf:params:jmap:submission"
   val JAMES_QUOTA: CapabilityIdentifier = "urn:apache:james:params:jmap:mail:quota"
   val JAMES_SHARES: CapabilityIdentifier = "urn:apache:james:params:jmap:mail:shares"
+  val JMAP_FILTER: CapabilityIdentifier = "urn:apache:james:params:jmap:mail:filter"
```

Comment:
JMAP will be a linagora specific extension hence we should use `"urn:linagora:openpaas:params:jmap:mail:filter"`